VideoApi/VideoApi.Client/VideoApiClient.cs

```diff
@@ -2991,6 +2991,16 @@ public async System.Threading.Tasks.Task StartOrResumeVideoHearingAsync(System.G
                             return;
                         }
                         else
+                        if (status_ == 400)
```

Comment:
is there a typo here, an else is blocked right after an if? Or am I reading it wrong?

Reply:
This is autogenerated code

Reply:
oh I think its just an else if condition